requirements-dev.txt

```diff
@@ -4,6 +4,7 @@ ethereum>=1.6.1,<2.0
 flaky>=3.3.0
 flake8==3.4.1
 hypothesis>=3.31.2
+lru-dict>=1.1.6
```

Comment:
this requirement is already present in `setup.py`.  You just need to re-install the package locally to pickup the new requirement.

Reply:
I'll remove this commit, glad I made it separately.